src/main/java/com/revaturelabs/ask/user/UserController.java

```diff
@@ -82,6 +91,34 @@ public User updateUser(@RequestBody User user, @PathVariable int id) {
     } catch (UserNotFoundException e) {
       throw new ResponseStatusException(HttpStatus.NOT_FOUND, "User not found", e);
     }
+    
+  }
+  
+  @PatchMapping("/profile/{id}")
+  public String updateUserInfo(MultipartHttpServletRequest user, @PathVariable int id) {
+
+    String key = "";
+    
+      try {
+        Part name = user.getPart("name");
+
+        MultipartFile image = user.getFile("myImage");
+        
+        User updatedUser = userService.findById(id);
+
+        key = userService.uploadProfilePicture(image, updatedUser.getUsername());
+        
+        updatedUser.setProfilePic(key);
+        userService.update(updatedUser);
+        
+      } catch (IOException e) {
+        throw new ResponseStatusException(HttpStatus.NO_CONTENT, "IOException on File.");
```

Comment:
NO_CONTENT is not an error response -- replace it with something appropriate please